rule-packs/mitre-attck-lateral-movement-attack-paths.json

```diff
@@ -0,0 +1,626 @@
+[
+    {
+      "name": "lateral-movement-exploitation-of-remote-services-patch-vulnerabilities-aws",
+      "description": "M1048 - Application Isolation and Sandboxing - Make it difficult for adversaries to advance their operation through exploitation of undiscovered or unpatched vulnerabilities by using sandboxing in AWS. Other types of virtualization and application microsegmentation may also mitigate the impact of some types of exploitation. Risks of additional exploits and weaknesses in these systems may still exist.",
+      "queries": [
+        {
+          "name": "query0",
+          "query": "FIND aws_instance THAT !HAS Finding",
+          "version": "v1"
+        }
+      ],
+      "alertLevel": "MEDIUM"
+    },
+    {
+        "name": "lateral-movement-exploitation-of-remote-services-patch-vulnerabilities-gcp",
+        "description": "M1048 - Application Isolation and Sandboxing - Make it difficult for adversaries to advance their operation through exploitation of undiscovered or unpatched vulnerabilities by using sandboxing in Google Cloud. Other types of virtualization and application microsegmentation may also mitigate the impact of some types of exploitation. Risks of additional exploits and weaknesses in these systems may still exist.",
+        "queries": [
+          {
+            "name": "query0",
+            "query": "FIND google_compute_instance THAT !HAS Finding",
```

Comment:
same as above, not a ton of value in the current J1QL

Reply:
Deleting for now.